[Docs][DocDB] Statistics-based full compactions documentation (#17667)
yugabyte/yugabyte-db: docs/content/preview/reference/configuration/yb-tserver.md

Default: `99`

##### --auto_compact_min_obsolete_keys_found

Minimum number of keys that must be read over the last `auto_compact_stat_window_seconds` to trigger a statistics-based full compaction.

Default: `10000`

##### --auto_compact_min_wait_between_seconds

Minimum wait time between statistics-based and scheduled full compactions. To be used if statistics-based compactions are triggering too frequently.

Default: `0`

##### --scheduled_full_compaction_frequency_hours

The frequency with which full compactions should be scheduled on tablets. `0` indicates that the feature is disabled. Recommended value: `720` hours or greater (i.e. 30 days).

Default: `0`

##### --scheduled_full_compaction_jitter_factor_percentage

Percentage of `scheduled_full_compaction_frequency_hours` to be used as jitter when determining full compaction schedule per tablet. Must be a value between `0` and `100`. Jitter is introduced to prevent many tablets from being scheduled for full compactions at the same time.

> Jitter will be deterministically computed when scheduling a compaction, between 0 and (frequency * jitter factor) hours. Once computed, the jitter will be subtracted from the intended compaction frequency to determined the tablet's next compaction time.

Reviewer: ```suggestion
Jitter is deterministically computed when scheduling a compaction, between 0 and (frequency * jitter factor) hours. Once computed, the jitter is subtracted from the intended compaction frequency to determine the tablet's next compaction time.
```
